freqtrade/freqtradebot.py

```diff
@@ -619,7 +623,7 @@ def _notify_buy(self, trade: Trade, order_type: str) -> None:
 
     def _notify_buy_cancel(self, trade: Trade, order_type: str, reason: str) -> None:
         """
-        Sends rpc notification when a buy cancel occurred.
+        Sends rpc notification when a buy/short cancel occurred.
```

Comment:
wouldn't something like the below make more sense (and keep wording simpler)

```suggestion
        Sends rpc notification when a entry order cancel occurred.
```